utils/math_functions.py

```diff
@@ -246,10 +246,10 @@ def stan_inverse(self,y,x):
 		return atan2(y,x)+3.14159265
 
 
-# def getPointBehindTheBall(point ,theta):
-#   x = point.x +(3.5 * BOT_RADIUS) *(math.cos(theta))
-#   y = point.y +(3.5 * BOT_RADIUS) *(math.sin(theta))
-#   return Vector2D(int(x), int(y))
+def getPointBehindTheBall(point ,theta, factor=3.5):
+  x = point.x +(factor * BOT_RADIUS) *(math.cos(theta))
```

Comment:
Indentation has been affected.

Reply:
@ultrainstinct30 , I think in your last PR of adding variable threshold for getbehindBall, the indentation got changed. Also it would be good to remove the redundant getPointbehindBall here : https://github.com/KRSSG/robocup/blob/603117038dd04967a9810d2fc5cf413f67d4e343/utils/math_functions.py#L214. Just have one function for this with variable threshold. It would be good to make a new PR only for this. Then edit this Pass receive PR.